streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamSessionWindowAggregate.java

```diff
@@ -75,22 +74,21 @@ public void enableSendingOldValues() {
         sendOldValues = true;
     }
 
-    private class KStreamSessionWindowAggregateProcessor extends AbstractProcessor<K, V> {
+    public class KStreamSessionWindowAggregateProcessor extends AbstractProcessor<K, V> {
```

Comment:
This class need to be accessed in the `instanceof` condition for lazy creation of sensors at NodeMetrics, ditto elsewhere.

Reply:
It's in `ProcessorNodeMetrics`.